internal/ethapi/api.coreth_test.go

```diff
@@ -0,0 +1,130 @@
+// (c) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package ethapi
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+type testSuggestPriceOptionsBackend struct {
```

Comment:
supernit perhaps rename it as a stub, i.e. `stubSuggestPriceOptionsBackend`

Reply:
I think that `testSuggestPriceOptionsBackend` aligns more closely with the existing `testBackend`. (I know in avalanchego we would definitely call this a `testXXX` but not sure what the pattern is in coreth.